bitnami/dataplatform-bp1/templates/_helpers.tpl

```diff
@@ -0,0 +1,65 @@
+{{/*
```

Comment:
Remove unused helpers

```
helper dp.kafka.fullname defined is not used anywhere in the yaml
helper dp.kafka.image defined is not used anywhere in the yaml
helper dp.solr.exporter-name defined is not used anywhere in the yaml
helper dp.solr.secretName defined is not used anywhere in the yaml
helper dp.spark.master.service.name defined is not used anywhere in the yaml
```

Reply:
That's true thanks, this file is no more needed